testdata/p4_14_samples_outputs/copy_to_cpu-first.p4

```diff
@@ -73,7 +77,7 @@ control egress(inout headers hdr, inout metadata meta, inout standard_metadata_t
 
 control ingress(inout headers hdr, inout metadata meta, inout standard_metadata_t standard_metadata) {
     @name(".do_copy_to_cpu") action do_copy_to_cpu() {
-        clone3<tuple<standard_metadata_t>>(CloneType.I2E, 32w250, { standard_metadata });
+        clone_preserving_field_list(CloneType.I2E, 32w250, (bit<8>)FieldLists.copy_to_cpu_fields);
```

Comment:
Does this implementation work to preserve standard metadata fields?  If yes, how does that work, if one cannot add annotations to the standard metadata struct definition?

Reply:
Regardless of the answer, it seems worth adding a sentence or three on this in the v1model.p4 comments that document how this works.

Reply:
You cannot. The idea is that you should not be able to spoof standard Metadata. It's debatable, but I don't know how to do it anyway.

Reply:
Seems like a reasonable restriction, and I will put it on a list of details to document elsewhere, including a suggested workaround that of course one is welcome to copy a standard metadata field value into a user-defined metadata field, and preserve the user-defined metadata field.